server/src/operators/dataset_operator.rs

```diff
@@ -256,6 +288,12 @@ pub async fn delete_dataset_by_id_query(
             break;
         }
 
+        let old_count = chunk_metadata_columns::chunk_metadata
+            .count()
+            .get_result::<i64>(&mut conn)
+            .await
+            .unwrap();
+
```

Comment:
I think this and the following count should be removed?

Reply:
ah, residue from the logging I was doing!